bin/boilerplate.rs

```diff
@@ -7,12 +7,14 @@ extern crate vangers;
 
 use vangers::{config, render};
 
-pub use self::glutin::{ElementState, KeyboardInput, ModifiersState, VirtualKeyCode as Key};
+pub use self::glutin::{ElementState, KeyboardInput, ModifiersState, VirtualKeyCode as Key, MouseScrollDelta, MouseButton};
 
 
 pub trait Application<R: gfx::Resources> {
     fn on_resize<F: gfx::Factory<R>>(&mut self, render::MainTargets<R>, &mut F);
     fn on_key(&mut self, KeyboardInput) -> bool;
+    fn on_mouse_wheel(&mut self, MouseScrollDelta);
```

Comment:
let's have default implementations that do nothing for those mouse events, given that only one of the apps need them atm